packages/stream_video_flutter/lib/src/call_screen/call_content/call_content.dart

```diff
@@ -113,6 +113,20 @@ class _StreamCallContentState extends State<StreamCallContent>
       }
     }
 
+    // Disable PiP when screen sharing is enabled
+    if (widget.callState.localParticipant?.isScreenShareEnabled !=
+        oldWidget.callState.localParticipant?.isScreenShareEnabled) {
+      if (widget.pictureInPictureConfiguration
+          .disablePictureInPictureWhenScreenSharing) {
+        StreamVideoFlutterBackground.setPictureInPictureEnabled(
+          enable: widget.pictureInPictureConfiguration.enablePictureInPicture &&
+              !(widget.callState.localParticipant?.isScreenShareEnabled ??
+                  false),
+        );
+      }
+    }
```

Comment:
Why is this? I think on the native iOS demo app we can have pip and screen sharing at the same time.